chore: update testkube integration docs (#3414)
kubeshop/tracetest: docs/docs/concepts/environment-token.mdx

---
id: environment-tokens
title: Environment Tokens
description: Environment Tokens are strings used to authenticate Tracetest CLI against an organization and environment.
hide_table_of_contents: false
keywords:
  - tracetest
  - trace-based testing
  - observability
  - distributed tracing
  - testing
image: https://res.cloudinary.com/djwdcmwdz/image/upload/v1698686403/docs/Blog_Thumbnail_14_rsvkmo.jpg
---

Environment Tokens are strings used to authenticate Tracetest CLI against an organization and environment.

![](https://res.cloudinary.com/djwdcmwdz/image/upload/v1701296331/docs/environment-tokens_gkt9kx.png)

> These tokens ared used to configure Tracetest CLI in environment that you cannot login with user interaction, like CI/CD environments.

Reviewer: ```suggestion
These tokens are used to configure the Tracetest CLI in environment where you cannot login with user interaction, like CI/CD environments.
```
